trace/trace.go

```diff
@@ -98,7 +98,11 @@ func (t *Tracer) Finish() string {
 
 	var strs []string
 	for _, s := range stages {
-		strs = append(strs, fmt.Sprintf("%q took %s", s.key, s.end.Sub(s.start).String()))
+		if s.end.After(time.Time{}) {
```

Comment:
If stage is unfinished, end time should be `time.Now()`

Reply:
No, this fixes exactly that. time.Time{} produces not now(), but Time(0), meaning end is not initialised.

Reply:
loose ≠ lose